packetbeat/protos/http/http.go

```diff
@@ -510,6 +516,18 @@ func (http *HTTP) publishTransaction(event common.MapStr) {
 }
 
 func (http *HTTP) collectHeaders(m *message) interface{} {
+
+	hdrs := map[string]interface{}{}
+
+	if !http.parserConfig.SendHeaders {
+		// not send all the headers
+		hdrs["content_length"] = m.ContentLength
```

Comment:
This might mean that if some headers are configured (e.g. `send_headers: ["host"]`), content_length and content_type are not sent, which I think is not what you want.